packages/ui/src/theme/tokens/components/fieldControl.ts

```diff
@@ -133,33 +149,24 @@ export const fieldcontrol: Required<FieldControlTokens<'default'>> = {
     borderBlockStart: { value: 'none' },
     borderRadius: { value: '0' },
     _focus: {
-      borderBlockEndColor: { value: '{colors.border.focus.value}' },
+      borderBlockEndColor: { value: 'transparent' },
```

Comment:
The quiet variation for inputs looks weird with the focus indicator + a bottom border, so this makes the bottom border transparent on focus.